staker-rewards/README.adoc

```diff
@@ -102,7 +102,7 @@ to a beneficiary.
 To calculate operator rewards and calculate merkle tree, drop option `-v`, and run:
 
 ```
-./rewards-merkle-generator.sh -w <eth_host> -s <start_timestamp> -e <end_timestamp> -r <reward_allocation>
+./rewards-merkle-generator.sh -w <eth_host> -s <start_timestamp> -e <end_timestamp> -x <start_block> -y <end_block> -r <reward_allocation>
```

Comment:
Flag names are getting really confusing. `-v` which traditionally stands for `version` is `verify`, `-x` and `-y` looks like two randomly-picked character. I think it's a good moment to take a step back and revisit flag names. I propose to use long flag names instead:

- `-v` -> `--verify`
- `-s` -> `--start-timestamp`
- `-e` -> `--end-timestamp`
- `-x` -> `--start-block`
- `-y` -> `--end-block`
- `-r` -> `--allocation`